src/Communication/Jaeger.Communication.Grpc/Gogoproto/Gogo.cs

```diff
@@ -0,0 +1,291 @@
+// <auto-generated>
```

Comment:
Kinda sad that this is being generated into the library, I was hoping gogo annotations, since they only apply to Go generator, would be no-op elsewhere.

Reply:
Sadly not. And it's not even possible to generate it without manually removing them. That was the first approach... But hard to maintain.